cfgov/legacy/templates/hud/housing_counselor_pdf_selfcontained.html

```diff
@@ -0,0 +1,281 @@
+{% load staticfiles %}
```

Comment:
Yes, it should be, mainly the reason I did this as a single self-contained file is to make it easier for the downstream wkhtmltopdf to process. See also GHE#989 under "PDF self-contained template".